static_src/components/users.jsx

```diff
@@ -230,11 +234,16 @@ export default class Users extends React.Component {
   get userParentEntityUserSelector() {
     // only show something if in a space and if the user is a org manager
     // or space manager.
-    if (!this.isSpace ||
-      (!this.currentUserIsSpaceManager && !this.currentUserIsOrgManager)) {
+    if (!this.isSpace) {
       return null;
     }
 
+    if (!this.currentUserIsOrgManager) {
```

Comment:
could this be `!this.currentUserIsOrgManager && !this.currentUserIsSpaceManager`?